spec/api/holdings_api_spec.rb

```diff
@@ -213,17 +213,17 @@ def parse_body(call)
 
   describe "v1/item_held_by" do
     it "returns an application error if the item_id does not match an ht_item" do
-      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1, organization: "umich")))
+      response = parse_body(v1(url_template("item_held_by", item_id: item_id_1)))
```

Comment:
I think I made this change on another branch as well so you may end up with a merge conflict